We ship netkit rsh, which isn't smart enough to deal with the oddball case
where argv[0] is an option flag.

--- krb5-1.3/src/appl/bsd/krsh.c
+++ krb5-1.3/src/appl/bsd/krsh.c
@@ -616,8 +616,10 @@
     else
       host = argv[0];
     
+#ifdef BERKELEY_RSH
     if (!strcmp(host, "rsh"))
       argv++;
+#endif
     
     fprintf(stderr,"trying normal rsh (%s)\n",
 	    UCB_RSH);
